### DISCUSSION TITLE
Installing / updating netdata over SSH produces error

### DISCUSSION THREAD

**Discussion by mstgrv** [Homebrew/discussions#4989] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.25-91-geb5f7fb
ORIGIN: https://github.com/Homebrew/brew
HEAD: eb5f7fb32b0477d2746a9da8500b1ac255f297ae
Last commit: 2 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 0e136c3220abef95bcffc06b4e638affe59426c0
Core tap last commit: 2 hours ago
Core tap branch: master
Core tap JSON: 18 Dec 02:06 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.2-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

When trying to install or upgrade netdata via homebrew over SSH, it always completes with the message:

> To restart netdata after an upgrade:
>   brew services restart netdata

When I run that command though, I get the following error:

> Warning: running over SSH, using user/* instead of gui/* domain!
> Hide this warning by setting HOMEBREW_SERVICES_NO_DOMAIN_WARNING.
> Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).

At this point, the commandline just hangs and does nothing until I Ctrl+C to quit. Is there any way to fix this so netdata is able to run over SSH?

**Comment by ctilley83** [comment]
They're not clear where to set that flag are they?

**Comment by wq9578** [comment]
This warning always appears if you run a `brew services` command via an SSH connection and has nothing to do with `brew services restart netdata` hanging. Submit a bug report if the problem persists despite upgrading.

The current warning message (e.g., just for the command `brew services`) is:
```
Warning: running over SSH without /dev/console ownership, using user/* instead of gui/* domain!
Hide this warning by setting HOMEBREW_SERVICES_NO_DOMAIN_WARNING.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
```

To hide the warning run `HOMEBREW_SERVICES_NO_DOMAIN_WARNING=1 brew services` or `HOMEBREW_SERVICES_NO_DOMAIN_WARNING=1 brew services restart netdata`.
